subprojects/core-api/src/main/java/org/gradle/api/artifacts/transform/InputArtifactDependencies.java

```diff
@@ -29,8 +29,30 @@
 /**
  * Attached to a property that should receive the <em>artifact dependencies</em> of the {@link InputArtifact} of an artifact transform.
  *
- * The order of the files match that of the dependencies in the source artifact view.
- * The type of the injected dependencies is {@link org.gradle.api.file.FileCollection}.
+ * <p>
+ *     The order of the files match that of the dependencies in the source artifact view.
+ *     The type of the injected dependencies is {@link org.gradle.api.file.FileCollection}.
+ *     For example, when a project depends on the guava jar, when the project is transformed (i.e. the project is the input artifact),
```

Comment:
"the guava jar" is not necessarily something people know. Maybe something that has transitive dependencies would be a good example. BTW, Guava since 24 does have its own dependencies, we now even ship one of them with Gradle (but it's complicated, so something else like SLF4J might be a good option).

Also, I think it should be spelled "Guava JAR", `guava.jar`, or perhaps "Guava jar".

Reply:
I am now using `spring-web` as an example.